ch4/control.go

```diff
@@ -20,13 +20,13 @@ func main() {
 	for _, num := range randoms {
 		switch {
 		case num%2 == 0 && num%3 == 0:
-			fmt.Println("Six!")
+			fmt.Println(num, " ", "Six!")
```

Comment:
This works, but as expressions get more complicated, you might want to look into using `fmt.Printf`:

```
fmt.Printf("%d Six!\n",num)
```

With a single variable, it's fine to just use `fmt.Println`